README.md

```diff
@@ -26,19 +26,17 @@ Eppo is a modular flagging and experimentation analysis tool. Eppo's SDKs are bu
 
 ### Release process
 
-To release a new version of SDK:
-1. Make sure that version strings have been updated:
-   - Eppo core: [eppo_core/Cargo.toml](eppo_core/Cargo.toml)
-   - Rust: [rust-sdk/Cargo.toml](rust-sdk/Cargo.toml)
-   - Python: [python-sdk/Cargo.toml](python-sdk/Cargo.toml)
-   - Ruby: [ruby-sdk/lib/eppo_client/version.rb](ruby-sdk/lib/eppo_client/version.rb) and [ruby-sdk/ext/eppo_client/Cargo.toml](ruby-sdk/ext/eppo_client/Cargo.toml)
+To release new versions:
```

Comment:
Just watched this video https://www.youtube.com/watch?v=puQYAhpfpkA to get a sense for how this would fit into our workflow, very interesting! I understand what you are saying on the PR comment that this doesn't require developers to be very strict w/ commits and we can just "be good once" at the release step.

I am open to trying this tool out for a few months.

I also understand @felipecsl 's feedback about better commit messages - that is absolutely something we should be as well, separate from the tool we use. Would you suggest a linter rule to enforce/remind this?